tests/wc-admin-helper.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * WCAdminHelper tests
+ *
+ * @package WooCommerce\Admin\Tests\WCAdminHelper
+ */
+
+use \Automattic\WooCommerce\Admin\WCAdminHelper;
+
+/**
+ * WC_Admin_Tests_Admin_Helper Class
+ *
+ * @package WooCommerce\Admin\Tests\WCAdminHelper
+ */
+class WC_Admin_Tests_Admin_Helper extends WP_UnitTestCase {
+
+	/**
+	 * Test wc_admin_active_for one hour
+	 */
+	public function test_is_wc_admin_active_for_one_hour() {
+		update_option( WCAdminHelper::WC_ADMIN_TIMESTAMP_OPTION, time() - ( HOUR_IN_SECONDS * 10 ) );
+
+		// Active for one hour - true.
+		$active_for = WCAdminHelper::is_wc_admin_active_for( HOUR_IN_SECONDS );
+		$this->assertEquals( true, $active_for );
+	}
+
+	/**
+	 * Test wc_admin_active_for 7 days
+	 */
+	public function test_is_wc_admin_active_for_7_days() {
+		update_option( WCAdminHelper::WC_ADMIN_TIMESTAMP_OPTION, time() - ( HOUR_IN_SECONDS * 10 ) );
+		// Active for 7 days - false.
+		$active_for = WCAdminHelper::is_wc_admin_active_for( DAY_IN_SECONDS * 7 );
+		$this->assertEquals( false, $active_for );
+	}
+
+	/**
+	 * @dataProvider range_provider
```

Comment:
🎉 This data provider is nice!

Reply:
Agreed, this is the first time I've seen it :-)

Reply:
Yeah it took me a little while to find an example, as it is not used often, but nice indeed :)